lib/Config/Populator.php

```diff
@@ -554,20 +552,6 @@ private function createSourceForSubscribers() {
     );
   }
 
-  private function updateNewsletterCategories() {
-    global $wpdb;
-    // perform once for versions below or equal to 3.14.0
-    if (version_compare($this->settings->get('db_version', '3.14.1'), '3.14.0', '>')) {
-      return false;
-    }
-    $query = "UPDATE `%s` SET categories = REPLACE(REPLACE(categories, ',\"blank\"', ''), ',\"sample\"', ',\"all\"')";
```

Comment:
This query removed `blank` and replaced `sample` with `all`. Now that we create the `blank` category, what happens to templates having the `sample` category? Wouldn't it make sense to replace `sample` by `blank`?

Reply:
> Wouldn't it make sense to replace `sample` by `blank`?

Definitely not. That would show all the templates in the blank section. That is not the point. We don't want to show all templates there, only the simple text ones.